airflow-core/tests/unit/jobs/test_triggerer_job.py

```diff
@@ -1145,3 +1145,14 @@ def test_update_triggers_prevents_duplicate_creation_queue_entries_with_multiple
     trigger_ids = {trigger.id for trigger in supervisor.creating_triggers}
     assert trigger_orm1.id in trigger_ids
     assert trigger_orm2.id in trigger_ids
+
+
+@patch("airflow.sdk.log.configure_logging")
+def test_process_log_messages_configures_logging(mock_configure_logging, supervisor_builder):
+    supervisor = supervisor_builder()
+
+    # Create the generator (this triggers the configuration reading)
+    log_processor_gen = supervisor._process_log_messages_from_subprocess()
+    next(log_processor_gen)  # Initialize the generator
+
+    assert mock_configure_logging.call_count == 1
```

Comment:
This test is overly tied to the implementation. As much as it pains me, I think this would be better without a unit test covering it.

Reply:
Yeah, fair enough, happy to drop it!

Reply:
done